iot-hub/Quickstarts/SimulatedDevice/nuget.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Might as well put this file at the root so it serves all projects in this repo.

Reply:
Good by me, if it would still work like the documentation describes (going into iot-hub\Quickstarts\SimulatedDevice directory and run dotnet restore).  I'm not familiar with how dotnet restore works so I assumed it needed to be in the same directory as Program.cs & SimulatedDevice.csproj.

Reply:
In my experience, that is how it worked. Here are the docs.

<https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#config-file-locations-and-uses>

> Current folder (aka Solution folder) or any folder up to the drive root.

Reply:
You are correct (just tried it). Should I close this PR and open a new one with the file at /azure-iot-samples-csharp/ (that's where I dropped the file & tested)?

Reply:
You can update your fork's branch and the PR will automatically update, or if you prefer you can abandon this PR and generate another. Either is fine with  me.

I appreciate you identifying the issue and helping with this change, though. Thanks!

Reply:
Thank you for your help on this @drwill-ms. I've edited my fork to have the nuget.config in the root (where I tested this yesterday). I didn't see it in the location I originally had on the PR so I think this will work as expected. If not please let me know.

Reply:
Did you push to remote? I'm not seeing the change it the files list.

Reply:
I just opened a new PR. I'm not editing from local repo on my machine so I'm not seeing how to do a push via the website. Sorry for the trouble.

Reply:
No problem :)